docs/gathered-data.md

```diff
@@ -421,6 +421,19 @@ Response see https://docs.openshift.com/container-platform/4.6/rest_api/workload
   * 4.7+
 
 
+## PNCC
+
+collects a summary of failed PodNetworkConnectivityChecks.
+The checks are requested via a dynamic client and
+then unmarshaled into the appropriate structure.
```

Comment:
It would be good to mention that only failed checks or `Failures` are collected.

Reply:
@tremes Well, doesn't it already say this?
> collects a summary of **failed** PodNetworkConnectivityChecks.

Reply:
Ahh OK I am blind. Ignore me :)